app/views/layouts/application.html.slim

```diff
@@ -5,5 +5,7 @@ html
     = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true
     = javascript_include_tag 'application', 'data-turbolinks-track' => true
     = csrf_meta_tags
-  body
+  body class=page_title
```

Comment:
@AnNOtis Could you open a new Pull Request to fix this? Thanks and good catch :+1: